src/pybind/mgr/dashboard/frontend/src/app/ceph/shared/ceph-shared.module.ts

```diff
@@ -1,7 +1,12 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
+import { DataTableModule } from '../../shared/datatable/datatable.module';
+import { SharedModule } from '../../shared/shared.module';
+import { DeviceListComponent } from './device-list/device-list.component';
 
 @NgModule({
-  imports: [CommonModule]
+  imports: [CommonModule, DataTableModule, SharedModule],
```

Comment:
```suggestion
  imports: [CommonModule, SharedModule],
```
`DataTableModule `is already imported and exported by `SharedModule`.
Wonder if we should also export `CommonModule`

Reply:
:man_shrugging: I have just adapted the code as much as it was required to get it running. Don't know if we should also export `CommonModule`.